src/components/Button/_button.scss

```diff
@@ -119,7 +119,7 @@
   text-align: center;
 }
 
-.cc-info-box .cc-cta {
+.c-info-box .cc-cta {
```

Comment:
This is all very confusing isn't it. `cc-cta` is only applied to JSX in corporate-react so I suggest moving everything below line 116 in this file to corporate-react. It should really be refactored as `.c-cta` too.